docs/pages/attributesJSON.rst

```diff
@@ -177,6 +174,12 @@ Below are object-specific physical quantities.  These values will override simil
 "requires_lighting"
 	- boolean
 	- Whether the object should be rendered with lighting or flat shading.
+"is_visible"
+	- boolean
+	- Whether the object's render asset should be created upon instancing.
```

Comment:
Functionality question : Can this be toggled after instancing, from the object's perspective? (i.e. can render assets be programmatically "turned on" and "turned off"?)

Reply:
In the TODO list of the PR description. Not currently implemented.